### DISCUSSION TITLE
thanhle.blog/nam-2023-cua-toi

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#91] [Ideas]
# thanhle.blog/nam-2023-cua-toi

Recap/Review năm 2023 của mình

https://thanhle.blog/blog/nam-2023-cua-toi

**Comment by khiem20tc** [comment]
Chúc anh Thành cùng team năm mới được khai lòng mở trí, sức lực vững vàng để tận hưởng và vượt qua các thử thách thú vị. Hi vọng cho một năm con rồng sẽ bớt khắc nghiệt và dễ thở hơn với anh em ngành IT mình tí :raised_hands:

**Comment by hai123vn** [comment]
anh ơi em đang thấy mấy đồng điện tử giá đang lên rất cao là do gì vậy anh, anh có thể ra 1 bài viết nói về vấn đề này không ?
